### DISCUSSION TITLE
无法获取公众号支付结果回调

### DISCUSSION THREAD

**Discussion by ElQut** [w7corp/easywechat#2550] [General]
easywechat 6.7 & ThinkPHP 6
无法获取公众号支付结果回调
![WechatIMG3274](https://user-images.githubusercontent.com/35054752/180127991-48c1ee82-47f5-437f-9bb6-f24c2aea95c6.png)

**Comment by overtrue** [comment]
麻烦认真看明白文档

ELQut ***@***.***>于2022年7月21日 周四12:08写道：

**Reply by ElQut** [reply]
谢谢大佬的提醒，我还是看不明白问题出在哪里，麻烦你说一下吗，谢谢，大佬

**Comment by overtrue** [comment]
你只是注册了server的处理逻辑，没有让server运行

ELQut ***@***.***>于2022年7月21日 周四12:31写道：
